src/test/scala/nl/knaw/dans/easy/bagindex/components/AddBagToIndexSpec.scala

```diff
@@ -25,13 +25,14 @@ class AddBagToIndexSpec extends BagIndexDatabaseFixture with AddBagToIndex {
 
   def addBaseTest(): UUID = {
     val bagId = UUID.randomUUID()
+    val doi = "10.5072/dans-x6f-kf66"
```

Comment:
I ended up making it a `lazy val` in the fixture.
Regarding the `bag-index.sql` file, I changed them to something else.